packages/marker-spiderfier/package.tpl.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
we don't have package.tpl.json files anymore. Instead, put stuff into package.json inside each package